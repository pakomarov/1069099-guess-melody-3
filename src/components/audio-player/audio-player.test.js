import React from 'react';
import renderer from 'react-test-renderer';
import AudioPlayer from './audio-player.jsx';


describe(`AudioPlayer rendering`, () => {
  it(`Should match snapshot of AudioPlayer when isPlaying is false`, () => {
    const tree = renderer
      .create(<AudioPlayer
        playerId={0}
        src={``}
        isPlaying={false}
        onToggle={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should match snapshot of AudioPlayer when isPlaying is true`, () => {
    const tree = renderer
      .create(<AudioPlayer
        playerId={0}
        src={``}
        isPlaying={true}
        onToggle={() => {}}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
