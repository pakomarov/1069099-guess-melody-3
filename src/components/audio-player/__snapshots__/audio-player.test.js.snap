// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer rendering Should match snapshot of AudioPlayer when isPlaying is false 1`] = `
Array [
  <button
    className="track__button track__button--play"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio
      onCanPlayThrough={[Function]}
      src=""
    />
  </div>,
]
`;

exports[`AudioPlayer rendering Should match snapshot of AudioPlayer when isPlaying is true 1`] = `
Array [
  <button
    className="track__button track__button--pause"
    disabled={true}
    onClick={[Function]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio
      onCanPlayThrough={[Function]}
      src=""
    />
  </div>,
]
`;
